service/service.go

```diff
@@ -87,6 +87,8 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 			}
 
 			esConfig.Transport = awsSignerRT
+		} else {
+			esConfig.Transport = dphttp.DefaultTransport
```

Comment:
Just a suggestion: how would you feel about adding this as a default value when creating the config in line 75 instead? It would then be overridden if required based on the signer flag